platform/source/mbed_rtc_time.cpp

```diff
@@ -52,7 +52,7 @@ static int _rtc_lpticker_isenabled(void)
 
 static time_t _rtc_lpticker_read(void)
 {
-    return _rtc_lp_timer->read_high_resolution_us() / US_PER_SEC + _rtc_lp_base;
+    return _rtc_lp_timer->elapsed_time().count() / US_PER_SEC + _rtc_lp_base;
```

Comment:
May as well let Chrono do the conversion. Should be no `US_PER_SEC` stuff remaining once Chrono is involved. I note `_rtc_lp_base` should be defined as `time_t`.

    return duration_cast<seconds>(_rtc_lp_timer->elapsed_time()).count() + _rtc_lp_base;